sys/include/crypto/aes.h

```diff
@@ -46,20 +46,35 @@ typedef uint8_t u8;
 
 #define AES_MAXNR         14
 #define AES_BLOCK_SIZE    16
-#define AES_KEY_SIZE      16
+
+/**
+ * @name AES key sizes
+ *
+ * The default key size is 128 bits. To use a different key size add
+ * USEMODULE += crypto_aes_192 and/ or USEMODULE += crypto_aes_256 to
+ * your Makefile.
+ *
+ * If only one key size is needed and that key size is not 128 bits, the 128 bit
+ * key size should be disabled with DISABLE_MODULE += crypto_aes_128 to enable
+ * an optimized build based on having a static key size.
+ *
+ * @{
+ */
```

Comment:
- Please move the documentation to the description of the module (top of this file).
- The 128 Bit key size **can** be disabled as an optimization.
- "based on having"? -> based on a static key size. Maybe give this paragraph an other read.